src/OverlayTrigger.js

```diff
@@ -281,19 +281,21 @@ const OverlayTrigger = React.createClass({
 
   _getContainerDimensions() {
     const containerNode = this.getContainerDOMNode();
-    let width, height;
+    let width, height, scroll;
+
     if (containerNode.tagName === 'BODY') {
       width = window.innerWidth;
       height = window.innerHeight;
+      scroll =
+        domUtils.ownerDocument(containerNode).documentElement.scrollTop ||
```

Comment:
I don't think the latter example works w/r/t the object shorthand.

As for the former example, it works but it feels a little wasteful - on the forward branch I explicitly know that I need to use `containerNode.scrollTop`, so that extra comparison seems unnecessary.